packages/manager/src/managers/project/ProjectManager.ts

```diff
@@ -50,9 +61,20 @@ export class ProjectManager extends BaseManager {
 				{ startDir: this.cwd },
 			);
 		} catch (error) {
-			throw new Error(
-				`Could not find a ${SLICE_MACHINE_CONFIG_FILENAME} file. Please create a config file at the root of your project.`,
-			);
+			const maybeDeprecatedConfigPath =
```

Comment:
It feels weird to have this one check in the managers and the other one in start-sm